system-variables.md

```diff
@@ -1109,11 +1109,11 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 >
 > Currently, this feature is not fully compatible with [altering multiple columns or indexes in a single `ALTER TABLE` statement](/sql-statements/sql-statement-alter-table.md). When adding a unique index with the index acceleration, you need to avoid altering other columns or indexes in the same statement.
 >
-> Currently, this feature is not compatible with [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md). When using index acceleration, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, including:
+> Currently, the performance of clusters with [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md) disabled is expected to be about 10 times faster than that in v6.1.0. However, there is no performance improvement for clusters with both PITR and index acceleration enabled. To optimize performance, it is recommended that you turn off PITR, add indexes in a quick way, then turn on PITR and perform a full backup. Otherwise, there are three expected behaviors:
```

Comment:
```suggestion
> Currently, the performance of clusters with [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md) disabled is expected to be about 10 times that in v6.1.0. However, there is no performance improvement when both PITR and index acceleration are enabled. To optimize performance, it is recommended that you disable PITR, add indexes in a quick way, then enable PITR and perform a full backup. Otherwise, three expected behaviors might occur:
```

Reply:
It is a little weird to say performance is 10 times faster than before. Instead, we can say the speed is x times faster.